src/lib.rs

```diff
@@ -18,7 +19,11 @@ pub fn init(app_id: Option<u32>) -> Result<(), Error> {
     let (steam_client, steam_single) = app_id
         .map(|app_id| Client::init_app(AppId(app_id)))
         .unwrap_or_else(Client::init)
-        .map_err(|e| Error::from_reason(e.to_string()))?;
+        .map_err(|e| match e {
```

Comment:
What is the purpose of this?

Reply:
After testing, this code can get more detailed init error reason.